common/templates/partial/_common_libs.html

```diff
@@ -0,0 +1,23 @@
+{% load static %}
+{% if sentry_dsn %}
+<script src="https://static.neodb.social/browser.sentry-cdn.com/7.7.0/bundle.min.js"></script>
+<script>
+    if (window.Sentry) Sentry.init({
+      dsn: "{{ sentry_dsn }}",
+      release: "NeoDB@{{ version_hash }}",
+      environment: "{{ settings_module }}",
+      tracesSampleRate: 1.0,
+    });
+</script>
+{% endif %}
+{% if jquery %}
+<script src="https://static.neodb.social/cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+{% else %}
+<script src="https://static.neodb.social/cdnjs.cloudflare.com/ajax/libs/cash/8.1.1/cash.min.js"></script>
```

Comment:
Can cash completely replace jquery in our use case?

Reply:
not yet user home editor, which I plan to fix later.